src/components/ErrorComponents/errorParser.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * This hook will take an error caught from a promise,
+ * and attempt to parse it and retrieve any message, code, and stacktrace
+ */
+
+// Passing in the intl object isn't ideal,
+// but this can't be a hook if it's going to run conditionally
+const errorParser = async (error, intl) => {
```

Comment:
Lets have default parm of `error = {}`  and destructure everything that we are pulling off of error here.

```suggestion
const { name, response, message } = error;
```